app/models/signature.rb

```diff
@@ -18,22 +18,25 @@
 #
 
 class Signature < ActiveRecord::Base
+
+  include EmailEncrypted
+  include SetPerishableToken
```

Comment:
I agree with @otlaitil - the names could be better - `EmailEncrypter` and `PerishableTokenGenerator` maybe?